main.cpp

```diff
@@ -160,7 +169,7 @@ int main(int argc, char** argv) {
 
     boost::asio::io_service::work work(ioService);
     processors = boost::thread::hardware_concurrency();
-
+    
```

Comment:
See http://vim.wikia.com/wiki/Remove_unwanted_spaces
What I use:
```
func! DeleteTrailingWS()
  exe "normal mz"
  %s/\s\+$//ge
  exe "normal `z"
endfunc
autocmd BufWrite *.{c,h,cpp,hpp,rs,java,py,etc...} :call DeleteTrailingWS()
```
The BufWrite glob can be modified depending on the files you use. Note that it isn't actually good to do this for all files. Try programming with [Whitespace](https://en.wikipedia.org/wiki/Whitespace_%28programming_language%29) when doing so for example (why you would actually use Whitespace is an entirely different question).

It also means that everything you touch gets whitespace stripped, and depending on what codebase you're working on, people can get annoyed if you start checking in changes where every file that you touch has loads of whitespace changes. I generally just use `git add -p` and avoid other people's whitespace issues.